### PR TITLE
[DOC] Warn about <flux:field.inline.fal> inside sections

### PR DESCRIPTION
**Pull request by cweiske**
See https://github.com/FluidTYPO3/fluidpages/issues/285

### CODE REVIEW

File: Classes/ViewHelpers/Field/Inline/FalViewHelper.php [hunk 1]
```diff
@@ -30,6 +30,21 @@
...
+ *
+ * Due to [TYPO3 core bug #71239](https://forge.typo3.org/issues/71239), using
+ * FAL references within sections (`<flux:form.section>`) in content elements
+ * or within the page configuration does not work.
+ *
+ * When choosing a file in one section element, you will see it in all sections.
+ * When choosing a file in a page configuration, it will be visible in the subpages
+ * configuration, too.
+ *
+ * This issue will most likely not fixed before TYPO3 8, so do not use it.
```
Comment (NamelessCoder) [inline Classes/ViewHelpers/Field/Inline/FalViewHelper.php]: I think a `be` got missed - `This issue will most likely not be fixed before TYPO3 8, so do not use it.`

File: Classes/ViewHelpers/Field/Inline/FalViewHelper.php [hunk 2]
```diff
@@ -30,6 +30,21 @@
...
+ *
+ * Alternatively, you could use `<flux:field.file>`.
+ *
+ *
```
Comment (NamelessCoder) [inline Classes/ViewHelpers/Field/Inline/FalViewHelper.php]: One too many blank lines ;)

**Comment by NamelessCoder** [comment]
I like this way of warning about the core bug - hopefully we will see a lot less people fall into this trap!

**Comment by cweiske** [comment]
Changed.

**Pull request merged**